aperturedb/sqlc/migrations/000003_lnc-sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS sessions (
+    id INTEGER PRIMARY KEY,
+
+    passphrase_entropy BLOB NOT NULL UNIQUE,
+    remote_static_pub_key BLOB UNIQUE,
```

Comment:
This can be `NULL` before or after the handshake is complete?

Reply:
only before the handshake is completed, when we create the session for first time (we do it before the connection)